src/conditional-compilation.md

```diff
@@ -299,6 +304,25 @@ r[cfg.proc_macro]
 Set when the crate being compiled is being compiled with the `proc_macro`
 [crate type].
 
+r[cfg.version]
+## The `version()` predicate
+
+r[cfg.version.behaviour]
+The `version()` predicate evaluates to true if both:
+
+* The version number contained inside follows the format and
+* The version number contained inside is less than or equal to the version
+  of the language the compiler targets. Usually the compiler version and
+  language version match. So compiler version `1.50.0` targets language
+  `1.50.0`.
```

Comment:
Usually the reference focuses on the language and not specifics about the compiler or specific implementations. The text here seems a little confusing in that regard. I would probably just strike this content.

```suggestion
  of the language the compiler targets.
```

Reply:
It might be beneficial to though to discuss a bit how this would work in alternative implementations like `mrustc` / `gccrs`? Though maybe "the language the compiler targets" is enough for that.